src/pages/aura/blockchains/eclipse.md

```
@@ -0,0 +1,13 @@
+---
+title: Aura on Eclipse SVM
+metaTitle: Aura on Eclipse SVM | Aura
+description: Learn about Metaplex Aura on the Eclipse SVM blockchain.
+---
+
+{% callout title="Aura Endpoints" %}
+
+**You can test out Aura on the Eclipse SVM at the following nodes:**
+- **Devnet**: https://aura-eclipse-devnet.metaplex.com 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)